lib/active_admin/views/pages/show.rb

```diff
@@ -25,7 +25,11 @@ def main_content
         end
 
         def attributes_table(*args, &block)
+          opts = args.extract_options!
           table_title = ActiveAdmin::Localizers.resource(active_admin_config).t(:details)
+          if given_title = opts[:title] || given_title = opts[:header]
```

Comment:
@ajw725 ,  looks like title is more preferable https://github.com/activeadmin/activeadmin/blob/9739ac47dc4dd7f821cdf84adf2dd3afed536942/lib/active_admin/views/pages/show.rb#L11